nimterop/cimport.nim

```diff
@@ -272,10 +316,9 @@ macro cAddStdDir*(mode = "c"): untyped =
   ##
   ## This allows something like this:
   ##
-  ## .. code-block:: nim
-  ##
-  ##    cAddStdDir()
-  ##    cImport cSearchPath("math.h")
+  runnableExamples:
+    cAddStdDir()
+    echo cSearchPath("math.h")
```

Comment:
`doAssert cSearchPath("math.h").len > 0` ?

Reply:
It's more documentation than a test.